<launch>
				<!--  ************** Odometry ***************  -->

	<node pkg="tf" type="static_transform_publisher" name="base_f_to_laser" args="0.0 0.0 0.25 0.0 0.0 0.0 /base_link /laser 30" />
    <!-- <node pkg="tf" type="static_transform_publisher" name="base_f_to_l" 
    args="0.0 0.0 0.0 3.14159 0.0 0.0 /base_footprint /base_link 40" /> -->
  <!-- <node name="lidar_port"  pkg="xv_11_laser_driver"  type="neato_laser_publisher" respawn="false" output="log">
    <param name="port"         type="string" value="/dev/ttyS0"/>  
    <param name="baudrate"     type="int"    value="115200"/>
    <param name="frame_id"     type="string" value="laser"/>
     <param name="rate"     type="int" value="100"/>
  </node> -->
 <node name="nox_controller" pkg="nox" type="nox_controller" output="screen">

      <param name="publish_tf" value="true" />
      <param name="robot_width" value="0.242" />
      <param name="publish_rate" value="20.0" />
      <param name="GEAR_RATIO" value="1.0" />
      <param name="print_data" value="false" />
      <param name="print_yaw" value="false" />
      <param name="use_mpu" value="false" />
      <param name="bias" value="0.0" />
      <param name="arduino" value='/dev/ttyUSB0' />
      <param name="baud" value="57600" />
      <param name="usePulseCount" value="0" />
      <param name="WHEEL_DIAMETER" value="0.043" />
      <param name="ENCODER_PULSES" value="720" />


	</node>	 
	<!-- <node name="pubVel" pkg="nox" type="pubVel.py" output="screen"/> -->
<!-- <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen" >
    <param name="fixed_frame" value = "odom"/>
     <param name="base_frame" value = "base_link"/>
    <param name="use_odom" value = "false"/>
    <param name="use_imu" value = "false"/>
    <param name="max_iterations" value="10"/>
    <param name="publish_pose" value="true"/>
    <param name="publish_tf" value="false"/>
    <param name="kf_dist_linear" value="0.05"/>
    <param name="kf_dist_angular " value="0.175"/>
  </node> -->
				<!--  ************** Sensors ***************  -->

	 <!-- <include file="/home/mecanum_robot/robot_ws/src/rplidar_ros/launch/rplidar_a1.launch"/> -->
	<!-- node name="rviz" pkg="rviz" type="rviz" args="-d $(find nox_description)/urdf.rviz" required="true" / -->
</launch>
